sdk/src/main/res/drawable/img_app_store_badge.xml

```diff
@@ -0,0 +1,109 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is copyright mandatory here like in the client? Asking due to tests failing on client when they're missing and it's pain in the ass to go back and add them again...and again and again

Reply:
It's not currently enforced in the SDK as we do not have checkstyle configured. Additionally, the SDK is open source so operates under a different licence. _Technically_ we should have a licence header at the top of each file but again, this is not currently enforced. We may (and ought to) add this in the future.